src/BenchmarkDotNet/Mathematics/StatisticalTesting/TostResult.cs

```diff
@@ -17,10 +17,10 @@ public TostResult(Threshold threshold, EquivalenceTestConclusion conclusion, [Ca
             FasterTestResult = fasterTestResult;
         }
 
-        public string ToStr(bool details) => details
-            ? ConclusionStr() + ": " + (SlowerTestResult?.PValueStr ?? "?") + "|" + (FasterTestResult?.PValueStr ?? "?")
-            : ConclusionStr();
+        public string ToPresentation(bool details) => details
```

Comment:
imho it would be better to call it `ToString`.

My main motivation is that so far "ToPresentation" name was always used by the code that generates boilerplate code. 

https://github.com/dotnet/BenchmarkDotNet/blob/2ba30330ec5ec8a9abe284c927277a6fe0fc5a0c/src/BenchmarkDotNet/Characteristics/CharacteristicPresenter.cs#L29

Reply:
Good catch! Fixed!